recipe/patch_yaml/snappy.yaml

```diff
@@ -1,8 +1,22 @@
 # from this snippet
 # _replace_pin('snappy >=1.1.7,<1.1.8.0a0', 'snappy >=1.1.7,<2.0.0.0a0', deps, record)
+#
+# Adapted with respect to the ABI change from 1.1.x to 1.2.x:
+# See: https://github.com/conda-forge/snappy-feedstock/issues/35
 if:
   name: "*"
 then:
   - replace_depends:
       old: snappy >=1.1.7,<1.1.8.0a0
-      new: snappy >=1.1.7,<2.0.0.0a0
+      new: snappy >=1.1.7,<1.2.0.0a0
+---
+# ABI change from 1.1.x to 1.2.x
+# See: https://github.com/conda-forge/snappy-feedstock/issues/35
+if:
+  name: "*"
+  has_depends: snappy
```

Comment:
This didn't catch anything, see the adjusted approach here: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/700